trillian/ctfe/ct_server/main.go

```diff
@@ -44,6 +45,7 @@ var (
 	metricsEndpoint    = flag.String("metrics_endpoint", "localhost:6963", "Endpoint for serving metrics; if left empty, metrics will be visible on --http_endpoint")
 	rpcBackendFlag     = flag.String("log_rpc_server", "localhost:8090", "Backend specification; comma-separated list or etcd service name (if --etcd_servers specified)")
 	rpcDeadlineFlag    = flag.Duration("rpc_deadline", time.Second*10, "Deadline for backend RPC requests")
+	getSTHInterval     = flag.Duration("get_sth_interval", time.Second*30, "Interval between internal get-sth operations (0 to disable)")
```

Comment:
30 seconds possibly too short if we have a large number of logs eventually? Will do for now.